cmd/kubeapps-apis/server/server.go

```diff
@@ -243,5 +245,41 @@ func gatewayMux() (*runtime.ServeMux, error) {
 		return nil, fmt.Errorf("failed to serve: %v", err)
 	}
 
+	svcRestConfig, err := rest.InClusterConfig()
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve in cluster configuration: %v", err)
+	}
+	coreClientSet, err := kubernetes.NewForConfig(svcRestConfig)
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve clientset: %v", err)
+	}
+
+	// TODO(rcastelblanq) Move this endpoint to the Operators plugin when implementing #4920
+	// Proxies the operator icon request to K8s
+	err = gwmux.HandlePath(http.MethodGet, "/operators/namespaces/{namespace}/operator/{name}/logo", func(w http.ResponseWriter, r *http.Request, pathParams map[string]string) {
```

Comment:
Operators logos are served now from here. Not the most elegant solution, but it should be something temporary.